posixlib/src/main/resources/scala-native/wordexp.c

```diff
@@ -55,6 +57,19 @@ _Static_assert(offsetof(struct scalanative_wordexp_t, we_offs) ==
 #define WRDE_SYNTAX -1
 #endif // _WIN32
 
+// OpenBSD needs dummy functions as well.
+#if defined(__OpenBSD__)
+
+#include <errno.h>
+
+int wordexp(const char *restrict s, void *restrict p, int flags) {
```

Comment:
Same as `strfmon_l`, we should define it as proxy/stub `scalanative_wordexp` and adjust bindings. Also applies to `wordfree` below.